db/schema.rb

```diff
@@ -59,6 +59,19 @@
     t.datetime "updated_at",       null: false
   end
 
+  create_table "loans", force: :cascade do |t|
+    t.string   "employee"
+    t.string   "item"
+    t.date     "dateBorrowed"
```

Comment:
very inconsistent here. why is this camelCase. Should be underscores in between words.